lib/utils.js

```diff
@@ -169,6 +169,17 @@ function getRuleExportsCJS (ast) {
     }, {});
 }
 
+/**
+ * Find the value of a property in an object by its property key name.
+ * @param {Object} obj
+ * @param {String} keyName
+ * @returns property value
+ */
+function findObjectPropertyValueByKeyName (obj, keyName) {
+  const property = obj.properties.find(prop => prop.key.type === 'Identifier' && prop.key.name === keyName);
+  return property ? property.value : undefined;
+}
+
```

Comment:
same here, better to have a test for it. :)

Reply:
Agreed, added a test for the new util function `collectReportViolationAndSuggestionData`.